cypress/support/pages/admin/pageAdministration.js

```diff
@@ -196,6 +196,14 @@ class Management {
 		cy.url().should("include", "/administration/courses");
 	}
 
+	navigateToNewCourseAdministration() {
+		cy.get(Management.#administrationOverviewNavigationButton).siblings()
+			.find(Management.#courseAdministrationNavigationButton)
+			.click()
+		//cy.get(Management.#courseAdministrationNavigationButton).click();
+		cy.url().should("include", "/administration/rooms/new");
```

Comment:
talk to mrika that /new is not a good url. maybe use the same pattern as the class overview.